tests/asset.rs

```diff
@@ -521,6 +527,72 @@ async fn allow_transfer_assets_to_one_beneficiary() -> Result<()> {
                 .await?;
                 debug!("Accept response: {:#?}", &accept_details);
                 assert!(issued_assets.contains(&accept_details.id), "RGB IDs match");
+
+                info!("First transfer worked! Attempting second...");
+
+                assets_vault_details = get_assets_vault(
+                    &tmp_vault.rgb_assets_descriptor_xpub,
+                    &tmp_vault.rgb_udas_descriptor_xpub,
+                )
+                .await?;
+
+                debug!("New assets vault details: {assets_vault_details:?}");
+
+                info!("Get a second Blinded UTXO");
+                debug!("First was: {blinded_utxo1:?}");
+                let blinded_utxo2 = get_blinded_utxo(&assets_vault_details.assets_change_output);
+                assert!(blinded_utxo2.is_err(), "TODO: Expected error in getting a second blinded UTXO because no new change output has been made");
+                return Ok(());
```

Comment:
This is correct?

Reply:
Yes, it was intentional, see the assertion on L544... Also be sure to understand the intention of this PR, as stated in the description.